ACKategories/Classes/Control+Blocks.swift

```diff
@@ -13,69 +13,113 @@ class CollectionWrapper<T: UIControl> {
 }
 
 class ActionWrapper<T: UIControl> {
-    let action: (T -> Void)
+    let action: ((T) -> Void)
 
-    init(action: (T -> Void)) {
+    init(action: @escaping ((T) -> Void)) {
         self.action = action
     }
 
-    @objc func invokeAction(sender: UIControl) {
+    @objc func invokeAction(_ sender: UIControl) {
         if let sender = sender as? T {
             action(sender)
         }
     }
 }
 
-public protocol UIControlEventHandling {
```

Comment:
Looks good, works bad 😞 I had to remove it because our nice way to have right type of "sender" seems not to work anymore in swift 3. So now we have "sender" only as UIControl :-/